components/Item/Card/Card.vue

```diff
@@ -7,7 +7,7 @@
   >
     <div
       class="card-box"
-      :class="{ 'card-margin': Margin, 'link-disabled': !link }"
+      :class="{ 'card-margin': Margin, 'link-disabled': link }"
```

Comment:
Shouldn't the L3 and L4 be edited?

Reply:
No we don't need to changes those.  Since we are just changing if `link`s default, not going from `noLink` to `link`